modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java

```diff
@@ -605,6 +612,26 @@ private InternetAddress[] _toInternetAddresses(String string)
 		return internetAddresses.toArray(new InternetAddress[0]);
 	}
 
+	private void _validateEmailAddress(String emailAddress)
+		throws PortalException {
+
+		String[] splitEmailAddresses = emailAddress.split(";");
+
+		if (splitEmailAddresses.length == 1) {
+			splitEmailAddresses = emailAddress.split(", ");
```

Comment:
@lucasmiranda0 we can also separate by comma, without the space. Please test this scenario and change this logic if necessary.

Reply:
@lucasmiranda0 I believe we can also separate by whitespace (space, tab, return, etc.). 

Could you confirm with the team the possible scenarios in the `#help-objects` slack channel?

Thanks!